pandas/tests/groupby/test_groupby.py

```diff
@@ -510,30 +510,30 @@ def test_frame_multi_key_function_list():
 
 
 @pytest.mark.parametrize('op', [lambda x: x.sum(), lambda x: x.mean()])
-@pytest.mark.filterwarnings("ignore:\\nPanel:FutureWarning")
 def test_groupby_multiple_columns(df, op):
     data = df
     grouped = data.groupby(['A', 'B'])
 
     result1 = op(grouped)
 
-    expected = defaultdict(dict)
+    keys = []
```

Comment:
This was a little bit more movement on the test than I had hoped for but the original test mentions something of this reshaping to be "a little bit crude" so it may be in order; new test should have the same coverage but with less reshaping